README.md

```diff
@@ -35,6 +35,24 @@ __options:__
 | :------------ |:-------------| :-----:|
 | filename     | the target filename path | - |
 
+## S3PersistStream
+
+Persists data that goes through the stream to a bucket in S3 and passes the raw data to the next stream
+
+### Usage:
+```javascript
+var S3PersistStream = require('bp-streams').S3PersistStream;
+var stream = new S3PersistStream(options);
+```
+
+__options:__
+
+| Name          | Description           | Default  |
+| :------------ |:-------------| :-----:|
+| metadata_version | filename | - |
```

Comment:
I would name the options here for what they would be use and not what values we will put in them, metadata_version is true for the specific case of metadata persist - but we can use the S3PersistStream also for other cases. The same about the stream_id.